modules/statics/src/account.ts

```diff
@@ -73,6 +73,7 @@ export interface EosCoinConstructorOptions extends AccountConstructorOptions {
 
 export interface SolCoinConstructorOptions extends AccountConstructorOptions {
   tokenAddress: string;
+  mintAddress: string;
```

Comment:
i think when we first added this, `tokenAddress` was meant to represent the mint pk.
don't need both properties, lets just keep 1 (no strong preference which).